http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20211015_support_proxy_to_azure_vm.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+<changeSet logicalFilePath="leonardo" author="qi" id="support_proxy_to_azure_vm">
+    <addColumn tableName="CLUSTER">
+        <column name="workspaceId" type="varchar(254)">
+            <constraints nullable="true"/>
+        </column>
+    </addColumn>
+
+    <addColumn tableName="CLUSTER">
+        <column name="cloudProvider" type="ENUM('GCP', 'AZURE')" defaultValue="GCP">
+            <constraints nullable="false"/>
+        </column>
+    </addColumn>
+
+    <addColumn tableName="PERSISTENT_DISK">
```

Comment:
Maybe would be nice to break up updates to different tables to different change sets? Just so if 1 fails it doesn't leave a table partially updated..

Reply:
I separated PD update and creating the new wsm resource table into a separate xml file.